mpas_analysis/ocean/ocean_regional_profiles.py

```diff
@@ -83,15 +86,29 @@ def __init__(self, config, regionMasksTask, controlConfig=None):
 
             fields = config.getexpression(regionGroupSection, 'fields')
 
+            max_bottom_depth = config.getexpression(regionGroupSection, 'maxDepth')
+            if max_bottom_depth:
+                max_bottom_depth = float(max_bottom_depth)
```

Comment:
```suggestion
            max_bottom_depth = config.getexpression(regionGroupSection, 'maxDepth')
```
There is no need to convert to a float here because `config.getexpression()` will do that already.